frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/utils.ts

```diff
@@ -107,3 +107,7 @@ export function replaceStringWithMaxLength(
 export function checkCommaInValue(str: string): string {
 	return str.includes(',') ? `"${str}"` : str;
 }
+
+export function getRemovePrefixAndOrder(tag: string): string {
+	return tag?.replace(/tag_|resource_|desc|asc/g, '').trim();
+}
```

Comment:
The replace here should only replace the `tag_`, `resource_` in the beginning and `asc`/`desc` in the end. When you make a change, think about where it could break. For example, say my tag is `description asc`; this will result in `ription` instead of `description`.

Reply:
@srikanthccv Removed replace logic: https://github.com/SigNoz/signoz/pull/2708/commits/8b7a7f7a8b774ff09e0fb0fbfae3ee205e185f70